test/kubemark/start-kubemark-master.sh

```diff
@@ -27,129 +237,204 @@ stdout_logfile=/var/log/${name}.log
 autorestart=true
 startretries=1000000
 EOF
+
+	# Update supervisord to make it run kubelet.
+	supervisorctl reread
+	supervisorctl update
 }
 
-EVENT_STORE_IP=$1
-EVENT_STORE_URL="http://${EVENT_STORE_IP}:4002"
-NUM_NODES=$2
-EVENT_PD=$3
-# KUBEMARK_ETCD_IMAGE may be empty so it has to be kept as a last argument
-KUBEMARK_ETCD_IMAGE=$4
-if [[ -z "${KUBEMARK_ETCD_IMAGE}" ]]; then
-  # Default etcd version.
-  KUBEMARK_ETCD_IMAGE="2.2.1"
-fi
-
-function retry() {
-	for i in {1..4}; do
-		"$@" && return 0 || sleep $i
-	done
-	"$@"
+# Create the log file and set its properties.
+#
+# $1 is the file to create.
+function prepare-log-file {
+	touch $1
+	chmod 644 $1
+	chown root:root $1
 }
 
-function mount-master-pd() {
-	local -r pd_name=$1
-	local -r mount_point=$2
-	if [[ ! -e "/dev/disk/by-id/${pd_name}" ]]; then
-		echo "Can't find ${pd_name}. Skipping mount."
-		return
+# Starts etcd server pod (and etcd-events pod if needed).
+# More specifically, it prepares dirs and files, sets the variable value
+# in the manifests, and copies them to /etc/kubernetes/manifests.
+function start-etcd-servers {
+  echo "Start etcd pods"
+  if [[ -d /etc/etcd ]]; then
+    rm -rf /etc/etcd
+  fi
+  if [[ -e /etc/default/etcd ]]; then
+    rm -f /etc/default/etcd
+  fi
+  if [[ -e /etc/systemd/system/etcd.service ]]; then
+    rm -f /etc/systemd/system/etcd.service
+  fi
+  if [[ -e /etc/init.d/etcd ]]; then
+    rm -f /etc/init.d/etcd
+  fi
+  prepare-log-file /var/log/etcd.log
+  cp "${KUBE_ROOT}/etcd.yaml" /etc/kubernetes/manifests
+
+  prepare-log-file /var/log/etcd-events.log
+  sed -i -e "s@{{instance_prefix}}@${INSTANCE_PREFIX}@g" "${KUBE_ROOT}/etcd-events.yaml"
```

Comment:
I was using `##` instead of `{{` - I'm fine with anything as long as it's consistent across kubemark codebase.